### PR TITLE
added actionbar

### PR DESCRIPTION
**Pull request by PuneetEsri**

### CODE REVIEW

File: arcgis-vectortiledlayer-url/src/main/java/com/esri/arcgisruntime/sample/arcgisvectortiledlayerurl/MainActivity.java [hunk 1]
```diff
@@ -124,4 +139,59 @@ private void selectItem(int position) {
...
+  @Override
+  public boolean onOptionsItemSelected(MenuItem item) {
+    // Handle action bar item clicks here. The action bar will
+    // automatically handle clicks on the Home/Up button, so long
+    // as you specify a parent activity in AndroidManifest.xml.
+    int id = item.getItemId();
+
+    // Activate the navigation drawer toggle
+    if (mDrawerToggle.onOptionsItemSelected(item)) {
+      return true;
```
Comment (doneill) [inline arcgis-vectortiledlayer-url/src/main/java/com/esri/arcgisruntime/sample/arcgisvectortiledlayerurl/MainActivity.java]: Can be simplified to `return mDrawerToggle.onOptionsItemSelected(item) || super.onOptionsItemSelected(item);`

File: arcgis-vectortiledlayer-url/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -1,5 +1,6 @@
 <resources>
     <string name="app_name">vector-tiled-layer</string>
+    <string name="action_settings">Settings</string>
```
Comment (doneill) [inline arcgis-vectortiledlayer-url/src/main/res/values/strings.xml]: `action_settings` never used, safe to remove

**Comment by doneill** [comment]
@PuneetEsri Some minor comments to address.

**Comment by PuneetEsri** [comment]
@doneill I have addressed your comments. Ok to merge?

**Comment by doneill** [comment]
@PuneetEsri Review complete, added zoom to an extent on start, merging now.

**Pull request merged**